app/sidekiq/central_mail/submit_form4142_job.rb

```diff
@@ -126,11 +126,16 @@ def upload_to_central_mail
     def upload_to_lighthouse
       @lighthouse_service = BenefitsIntakeService::Service.new(with_upload_location: true)
 
+      Rails.logger.info(
+        'Successful Form4142 Submission to Lighthouse',
+        { benefits_intake_uuid: @lighthouse_service.uuid, submission_id: submission.id }
```

Comment:
`@submission_id` has already been defined in the class. Could it be used here instead?